Clarify single hypertable in continuous aggregate (#509)
timescale/docs.timescale.com-content: api.md

         [ optional grouping exprs>]
[HAVING ...]
```
Note that continuous aggregates have some limitations of what types of
queries they can support, described in more length below.  For example,
the `FROM` clause must provide only one hypertable, i.e., no joins, views or 

> subqueries ares upported, the `GROUP BY` clause must include a time bucket on 

Reviewer: ```suggestion
subqueries are supported, the `GROUP BY` clause must include a time bucket on 
```

         [ optional grouping exprs>]
[HAVING ...]
```

> Note that a single hypertable is supported in the `FROM` clause.

Reviewer: ```suggestion
Note that continuous aggregates have some limitations of what types of
queries they can support, described in more length below.  For example,
the `FROM` clause must provide a single hypertable name (no JOINs or subqueries),
the `GROUP BY` must include a time bucket on the hypertable's time column, and
all aggregates must be parallelizable.
```

Author: Used the above text with some modifications.

> Note that only hypertable and only one is allowed in `FROM` clause.

Reviewer 2: Current sentence reads a bit strange. I suggest:
```suggestion
Note that only a single hypertable is supported in the `FROM` clause.
```

Author: I agree it is a little bit strange, but the suggestion might not emphasise `hypertable` strong enough. But I will go for it.

#### Parameters
|Name|Description|
|---|---|
| `<view_name>` | Name (optionally schema-qualified) of continuous aggregate view to be created.|
| `<column_name>`| Optional list of names to be used for columns of the view. If not given, the column names are deduced from the query.|
| `WITH` clause | This clause specifies [options](#continuous_aggregate-create_view-with) for the continuous aggregate view.|

> | `<select_query>`| A `SELECT` query that uses the specified syntax, which allows only a hypertable in `FROM` clause. |

Reviewer 2: ```suggestion
| `<select_query>`| A `SELECT` query that uses the specified syntax, which allows only a single hypertable in the `FROM` clause. |
```

> | `<select_query>`| A `SELECT` query that uses the specified syntax, which allows only a singe hypertable in the `FROM` clause. |

Reviewer: Let's not make this change.  Two reminders is enough, and very long parameter args just are hard to read.  Plus, the "suggestion" as written is actually confusing.

Author: Removed

- The view will be automatically refreshed (as outlined under
  [`refresh_continuous_aggregate`](#refresh_continuous_aggregate))
  unless `WITH NO DATA` is given (`WITH DATA` is the default).
- The `SELECT` query should be of the form specified in the syntax above.
- Only a single hypertable can be specified in the `FROM` clause of the 

>   `SELECT` query.

Author: Made longer clarifications

  [`refresh_continuous_aggregate`](#refresh_continuous_aggregate))
  unless `WITH NO DATA` is given (`WITH DATA` is the default).
- `SELECT` query should be of the form specified in the syntax above.

> - Only hypertable and only one can be specified in `FROM` clause of `SELECT`
>   query.

Reviewer 2: ```suggestion
- Only a single hypertable  can be specified in the `FROM` clause of a `SELECT`
  query.
```
